releasenotes/notes/add-elasticsearch-curator-88089d04f7ccd549.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Add Elasticsearch Curator for managing aggregated log data.
```

Comment:
More for upstream, we should probably say how to turn it on, and what it does if you turn it on.